apps/core/scenes/search/SearchContext.js

```diff
@@ -89,7 +91,7 @@ type State = {|
 const defaultDepartureDate = DateFNS.addDays(new Date(), 1);
 const defaultReturnDate = DateFNS.addDays(defaultDepartureDate, 2);
 const defaultSortBy = 'QUALITY';
-
+const defaultLimit = 50;
```

Comment:
Maybe we should move default values to `margarita-config`, what do you think?

Reply:
@jvaclavik done!